templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/terraform/main.tf

```diff
@@ -78,3 +78,15 @@ output "azure_resource_id" {
 output "connection_uri" {
   value = "https://${data.azurerm_app_service.guacamole.default_site_hostname}/?/client/${textencodebase64("${azurerm_windows_virtual_machine.windowsvm.name}\u0000c\u0000azuretre", "UTF-8")}"
 }
+
+output "vm_username" {
+  value = random_string.username.result
+}
+
+output "vm_password_secret_name" {
+  value = local.vm_password_secret_name
+}
+
+output "keyvault_name" {
```

Comment:
why do we need teh keyvault name?

Reply:
So that we can persist the new password to key vault - its not outputted currently and isn't set as a parameter so no other way to get it